idempotent-consumer/README.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
To be in line with other microservice patterns, we should change the top level folder name to `microservices-idempotent-consumer`

Reply:
done!